staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -469,7 +482,10 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 		preShutdownHooks:       map[string]preShutdownHookEntry{},
 		disabledPostStartHooks: c.DisabledPostStartHooks,
 
-		healthzChecks: c.HealthzChecks,
+		healthChecks:             c.HealthzChecks,
+		readinessChecks:          c.ReadyzChecks,
+		readinessStopCh:          make(chan struct{}),
```

Comment:
what does it do, document with a oneliner

Reply:
sure

Reply:
actually below in the type